sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare.py

```diff
@@ -0,0 +1,682 @@
+        "text_analytics_account": "https://cognitiveusw2dev.azure-api.net"
+    })
+    def test_bad_document_input(self, client):
+        docs = "This is the wrong type"
+
+        with self.assertRaises(TypeError):
+            response = client.begin_analyze_healthcare(docs)
+
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer(client_kwargs={
+        "api_version": TextAnalyticsApiVersion.V3_1_PREVIEW,
+        "text_analytics_account_key": os.environ.get('AZURE_TEXT_ANALYTICS_KEY'),
+        "text_analytics_account": "https://cognitiveusw2dev.azure-api.net"
+    })
+    def test_mixing_inputs(self, client):
+        docs = [
+            {"id": "1", "text": "Microsoft was founded by Bill Gates and Paul Allen."},
+            TextDocumentInput(id="2", text="I did not like the hotel we stayed at. It was too expensive."),
+            u"You cannot mix string input with the above inputs"
+        ]
+        with self.assertRaises(TypeError):
+            response = client.begin_analyze_healthcare(docs)
+
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer(client_kwargs={
+        "api_version": TextAnalyticsApiVersion.V3_1_PREVIEW,
+        "text_analytics_account_key": os.environ.get('AZURE_TEXT_ANALYTICS_KEY'),
+        "text_analytics_account": "https://cognitiveusw2dev.azure-api.net"
+    })
+    def test_out_of_order_ids(self, client):
+        docs = [{"id": "56", "text": ":)"},
+                {"id": "0", "text": ":("},
+                {"id": "22", "text": ""},
+                {"id": "19", "text": ":P"},
+                {"id": "1", "text": ":D"}]
+
+        response = list(client.begin_analyze_healthcare(docs).result())
+        expected_order = ["56", "0", "22", "19", "1"]
+        actual_order = [x.id for x in response]
+        print(actual_order)
+        for idx, resp in enumerate(response):
+            self.assertEqual(resp.id, expected_order[idx])
+
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer(client_kwargs={
+        "api_version": TextAnalyticsApiVersion.V3_1_PREVIEW,
+        "text_analytics_account_key": os.environ.get('AZURE_TEXT_ANALYTICS_KEY'),
+        "text_analytics_account": "https://cognitiveusw2dev.azure-api.net"
+    })
+    def test_show_stats_and_model_version(self, client):
+        docs = [{"id": "56", "text": ":)"},
+                {"id": "0", "text": ":("},
+                {"id": "22", "text": ""},
+                {"id": "19", "text": ":P"},
+                {"id": "1", "text": ":D"}]
+
+        response = client.begin_analyze_healthcare(
+            docs,
+            show_stats=True,
+            model_version="2020-09-03"
+        ).result()
+
+        self.assertIsNotNone(response)
+        self.assertIsNotNone(response.model_version)
+        self.assertEqual("2020-09-03", response.model_version)
+        self.assertEqual(response.statistics.documents_count, 5)
```

Comment:
would it be good to also check the document statistics?

Reply:
Yep, I've added this.